### DISCUSSION TITLE
How to create CustomAction project for net6 ?

### DISCUSSION THREAD

**Discussion by svg2003** [wixtoolset/issues#8013] [Questions]
Is there a way (or any plans to add ability), to use net6 for CustomAction project? 

Right now it will give below error:

EXEC : error : Invalid argument: WixToolset.Dtf.WindowsInstaller.dll must be included in the list of support files. If using the MSBuild targets, make sure the assembly reference has the Private (Copy Local) flag set.

And even if you copy DLL into your project, finally it will crash during the build (due to net472-based utility cannot load net6 dlls).

I found this thread and seems no possible standard workarounds.
https://stackoverflow.com/questions/76440520/wix4-c-sharp-customaction-project-compile-error-wixtoolset-dtf-windowsinstalle

**Comment by robmen** [comment]
Yes. This is something we are designing.
